{
	"id": "0ef3398e-da21-11ee-b23a-080027a5b8e9",
	"advisories": [
		{
			"content": {
				"id": "0ef3398e-da21-11ee-b23a-080027a5b8e9",
				"title": "Django -- multiple vulnerabilities",
				"description": "\n\t<p>Django reports:</p>\n\t<blockquote cite=\"https://www.djangoproject.com/weblog/2024/mar/04/security-releases/\">\n\t  <p>CVE-2024-27351: Potential regular expression denial-of-service in\n\t    django.utils.text.Truncator.words().</p>\n\t</blockquote>\n\t",
				"references": [
					{
						"source": "vuxml.freebsd.org",
						"url": "https://www.djangoproject.com/weblog/2024/mar/04/security-releases/"
					},
					{
						"source": "vuxml.freebsd.org",
						"url": "https://www.vuxml.org/freebsd/0ef3398e-da21-11ee-b23a-080027a5b8e9.html"
					}
				],
				"published": "2024-03-04T00:00:00Z"
			},
			"segments": [
				{
					"ecosystem": "freebsd"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2024-27351",
				"references": [
					{
						"source": "vuxml.freebsd.org",
						"url": "https://www.cve.org/CVERecord?id=CVE-2024-27351"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "freebsd"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "freebsd",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py310-django32"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "3.2.25"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py310-django42"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "4.2.11"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py310-django50"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "5.0.3"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py311-django32"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "3.2.25"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py311-django42"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "4.2.11"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py311-django50"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "5.0.3"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py39-django32"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "3.2.25"
										}
									]
								}
							},
							{
								"vulnerable": true,
								"fix_status": {
									"class": "unknown"
								},
								"package": {
									"name": "py39-django42"
								},
								"affected": {
									"type": "freebsd-pkg",
									"range": [
										{
											"lt": "4.2.11"
										}
									]
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "freebsd",
		"raws": [
			"vuls-data-raw-freebsd/0ef3398e-da21-11ee-b23a-080027a5b8e9.json"
		]
	}
}
